src/main/perf/IndexState.java

```diff
@@ -103,7 +103,11 @@ public synchronized SortedSetDocValuesReaderState getSortedSetReaderState(String
     SortedSetDocValuesReaderState result = ssdvFacetStates.get(facetGroupField);
     if (result == null) {
       IndexSearcher searcher = mgr.acquire();
-      result = new DefaultSortedSetDocValuesReaderState(searcher.getIndexReader(), facetGroupField);
+      if (facetsConfig != null) {
```

Comment:
Hmm was this a pre-existing bug?  Or none of our DV faceting cared about the FacetsConfig until now (since we now can express hierarchical or not)?

Reply:
Yeah passing in FacetsConfig to DV faceting is a new change. There's another PR here that also uses this feature: https://github.com/apache/lucene/pull/611/files